css-brochure-form/styles.css

```diff
@@ -0,0 +1,84 @@
+body {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  text-align: center;
+}
+
+header h1 {
+  font-size: 1.3rem;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: #ec5d57;
+}
+
+label {
+  font-size: 0.75rem;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+input {
+  width: 30rem;
+  height: 40px;
+  margin-top: 5px;
+  margin-bottom: 15px;
+  border: 1px solid #a6aaa9;
+  border-radius: 5px;
+}
+
+select {
+  width: 30rem;
+  height: 40px;
+  margin-top: 5px;
+  margin-bottom: 15px;
+  border: 1px solid #a6aaa9;
+  border-radius: 5px;
+}
+
+input::placeholder {
+  font-size: 0.9rem;
+  padding-left: 6px;
+  color: #aaa;
+}
+
+button {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.05rem;
+  color: white;
+  background-color: #ec5d57;
+  margin-top: 0;
+  margin-left: 22.5rem;
+  border-radius: 20px;
+  border: none;
+  padding: 7px 30px;
+  text-transform: uppercase;
+}
+
+.email > label {
```

Comment:
A couple things to consider. 
1. instead of having a class for .email, .name, .hiring, .engineer you might have a more generic class such as .form-control.

With CSS you could then target your labels just once:
```
.form-control > label {

}
```

Totally your choice if you want to do that. The other thing that I think you should change is having `margin-left` instead of `margin-right`. If the placeholder text changed you would also need to change the value for `margin-right` as you have it. 

Also, I would probably error on the side of using pixels for these values as you don't really need responsiveness over here as much as you need a fixed distance from the left side of the input.

Reply:
If you have any problem with `margin-left` you may want to try `padding-left`.